certified-connectors/Cireson Service Manager Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2242 @@
+                    "value-title": "Name",
+                    "parameters": {
+                      "Id": "56c99a7d-6ac7-ab3c-e6c0-bbf5fe76a65c"
+                    }
+                  },
+                  "title": "Status"
+                },
+                "AssignedWorkItem": {
+                  "description": "Assigned User",
+                  "type": "string",
+                  "x-ms-summary": "Assigned User",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetUsers",
+                    "value-path": "BaseId",
+                    "value-title": "Name"
+                  },
+                  "title": "Assigned User"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/{Location}": {
+      "delete": {
+        "responses": {
+          "204": {
+            "description": "Delete Webhook Trigger",
+            "schema": {}
+          }
+        },
+        "summary": "DeleteWebhook",
+        "description": "DeleteWebhook",
+        "operationId": "DeleteWebhook",
+        "parameters": [
+          {
+            "name": "Location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/platform/api/CreateWebhooks": {
+      "x-ms-notification-content": {
+        "schema": {
+          "$ref": "#/definitions/WorkItem_WebhookResponse"
+        }
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Created Webhook Trigger",
+            "schema": {
+              "$ref": "#/definitions/WebhookSettings"
+            }
+          }
+        },
+        "x-ms-trigger": "single",
+        "summary": "When a New Work Item is Created",
+        "description": "When a New Work Item is Created",
+        "operationId": "CreateWebhook",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "webhookSettings": {
+                  "type": "object",
+                  "properties": {
+                    "Url": {
+                      "type": "string",
+                      "description": "Url",
+                      "title": "Url",
+                      "x-ms-notification-url": true,
+                      "x-ms-visibility": "internal"
+                    },
+                    "WorkItemClassType": {
+                      "type": "string",
+                      "description": "Workitem Class Type",

```

Comment:
@CiresonOps please make sure that everything is grammatically correct and is free of typographical errors.